src/xmipp/libraries/data/mask.h

```diff
@@ -939,9 +939,9 @@ void compute_hist_within_binary_mask(const MultidimArray< int >& mask,
     }
 }
 
-#define COUNT_ABOVE 1
-#define COUNT_BELOW 2
-#define COUNT_BETWEEN 3
+constexpr int  COUNT_ABOVE = 1;
```

Comment:
same here (basically anywhere where you have multiple values to choose from, but you don't care about their value)